File: HienMau/app/src/main/java/com/example/hienmau/models/KetQuaHienMau.java
package com.example.hienmau.models;

import com.google.gson.annotations.SerializedName;

public class KetQuaHienMau {
    @SerializedName("XacNhanId")
    public int xacNhanId;

    @SerializedName("NgayHien")
    public String ngayHien;
    @SerializedName("NhomMau")
    public String nhomMau;

    @SerializedName("HeRh")
    public String heRh;

    @SerializedName("LuongMau")
    public int luongMau;

    @SerializedName("ChieuCao")
    public int chieuCao;

    @SerializedName("CanNang")
    public double canNang;

    @SerializedName("HuyetApTamThu")
    public int huyetApTamThu;

    @SerializedName("HuyetApTamTruong")
    public int huyetApTamTruong;
}

File: QuanLyHienMauApi/QuanLyHienMauApi/Dtos/KetQuaHienMau.cs
namespace QuanLyHienMauApi.Dtos
{
    public class KetQuaHienMau
    {
        public int XacNhanId { get; set; }
        public DateTime NgayHien { get; set; }
        public string NhomMau { get; set; }
        public string HeRh { get; set; }
        public int LuongMau { get; set; }

        public int ChieuCao { get; set; }
        public double CanNang { get; set; }
        public int HuyetApTamThu { get; set; }
        public int HuyetApTamTruong { get; set; }
    }
}
